src/custom/pages/Faq/index.tsx

```diff
@@ -415,6 +466,21 @@ export default function Faq() {
             allow you to wrap and unwrap ETH into WETH without needing to leave the dapp’s UI
           </p>
 
+          <h3 id="why-is-it-more-troublesome-to-trade-eth-to-erc20-than-erc20-to-eth">
+            Why is it more troublesome to trade ETH to ERC20 token, than ERC20 token to ETH?
+          </h3>
+
+          <p>
+            CowSwap needs to operate with WETH (wrapped Ether), and when wrapping and unwrapping it, trading amounts may
```

Comment:
I would explain it differently, I feel is not bad to do a longer explanation in this delicate topic:

CowSwap can only work with ERC20 tokens. ETH is the native Ethereum currency, which is not an ERC20 token.

In order to sell ETH, you need to wrap it first. Wrapping is done by doing a ETH deposit in WETH contract. When you deposit, you will get a balance of WETH of the deposited ETH amount. 

Any time, you can withdraw your ETH from the WETH contract. This is called unwrapping WETH.

Wrapping and unwrapping ETH are done using regular Ethereum transactions to WETH contract, which is not related to CowSwap. For CowSwap WETH is just one ERC20 token that can be sold or bought. 

Although CowSwap doesn't allow to sell ETH, it will help you do the wrapping, so you can easily turn ETH into WETH.

Although ETH cannot be sold, you can buy ETH. This is because CowSwap will allow you to buy WETH and it will unwrap it for you.

Reply:
I edited @anxolin 's comment and i think it should be included :) @avsavsavs